src/org/sosy_lab/java_smt/solvers/z3/Z3ToResolutionProofConverter.java

```diff
@@ -400,143 +400,143 @@ ProofNode handleTransitivityStar(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleRewriteStar(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handlePullQuant(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handlePullQuant(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleElimUnusedVars(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleElimUnusedVars(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handlePushQuant(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handlePushQuant(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleDer(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleDer(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleQuantInst(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleQuantInst(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleHypothesis(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleHypothesis(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleLemma(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleLemma(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleUnitResolution(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleUnitResolution(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleIffTrue(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleIffTrue(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleIffFalse(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleIffFalse(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleCommutativity(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleCommutativity(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleDefAxiom(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleDefAxiom(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleAssumptionAdd(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleAssumptionAdd(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleLemmaAdd(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleLemmaAdd(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleRedundantDel(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleRedundantDel(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleClauseTrail(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleClauseTrail(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleDefIntro(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleDefIntro(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleApplyDef(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleApplyDef(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleIffOeq(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleIffOeq(Z3ProofDAG.Z3ProofNode node) {
     return null;
   }
 
-  ProofNode handleNnfPos(Z3ProofDag.Z3ProofNode node) {
+  ProofNode handleNnfPos(Z3ProofDAG.Z3ProofNode node) {
     return null;
```

Comment:
It is generally not advised to return `null`. 
Just throw a `UnsupportedOperationException` then you will know which parts you still need to implement or inherit/override.